doc/api.rst

```diff
@@ -0,0 +1,24 @@
+API
+---
+
+.. autosummary::
+    :toctree: generated/
+
+    openmmml.MLPotential
+
+
+Models
+~~~~~~
+.. autosummary::
+    :toctree: generated/
+   
+    openmmml.models.anipotential.ANIPotentialImpl
```

Comment:
Let's move this down below the Developer API section.  It's conceptually the lowest level part.  The abstract superclasses should be listed before the concrete subclasses.

Is there a way we could make it automatically include all classes in the `models` module, so we don't have to update this file manually?

Reply:
You can use the autodoc extension. It does not allow to do that directly, but you can use the automatically generated file for the document to do so by using "include", like here:
https://github.com/torchmd/torchmd-net/blob/5ebae6560a8d59747fc8720fdbd9222cf8489716/docs/source/datasets.rst?plain=1#L74-L81

Reply:
Used jinja2 template + a python script to get the classes in models the same way as in the OpenMM docs